.bazelrc

```diff
@@ -1,5 +1,5 @@
 # Include git version info
-build --workspace_status_command hack/build/print-workspace-status.sh
+build --workspace_status_command tools/build/print-workspace-status.sh
```

Comment:
Perhaps the Bazel related things should all go into `build/bazel`? Including:

* Workspace dependency .bzl files
* print-workspace-status.sh
* .bazelrc

etc.

For those files that _have_ to be in the root of the repo (e.g. .bazelrc), we can use a symlink into this directory too. It'll keep things nice and organised into a single directory that way 😄